swap_meet/item.py

```diff
@@ -1,2 +1,26 @@
+import uuid 
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, category=""):
+        self.id = id if id is not None else uuid.uuid4().int
+        self.condition = condition
+        self.category = category
+    
+    def get_category(self):
+        return self.__class__.__name__
+    
+    def __str__(self):
+        return f"An object of type {self.__class__.__name__} with id {self.id}."
```

Comment:
Instead of doing `self.__class__.__name__`, we could use our `get_category` method to DRY up our code some. 
```suggestion
    def __str__(self):
        return f"An object of type {self.get_category} with id {self.id}."
```

Reply:
Ok! Modified the line to now read:

`    def __str__(self):
        return f"An object of type {self.get_category()} with id {self.id}."`